js/templates/item.html

```diff
@@ -41,6 +41,11 @@ <h2 class="page-contractTitle noOverflow textOpacity1 marginLeft20 marginBottom8
             <%= polyglot.t('BUYNOW') %>
           </div>
         </li>
+        <% if (ob.totalReviews) { %>
+        <li class="border0 alignCenter padding10 marginBottom12 js-itemRating">
+          <span class="marginRight3"><%= ob.totalReviews %> Review<% ob.totalReviews !== 1 && print('s') %></span><% print(ob.starsTemplate({ rating: ob.avgRating })) %><span class="marginLeft2">(<%= ob.avgRating %>/5)</span>
```

Comment:
I don't think Reviews with only one review would be too bad, it's a pretty typical thing. 

It would be nice to provide plurals, but that gets complicated for some languages that have different plural forms for different numbers (0, 1, 2-3, more than 4, etc.). We can do that if we pass the number back to polyglot, and set the language file up right. 

See more here: http://airbnb.io/polyglot.js/